src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,49 @@
+package mate.jdbc.model;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {

```

Comment:
you can create one more constructor here
```
public Driver(String name, String licenseNumber) {
        this.name = name;
        this.licenseNumber = licenseNumber;
    }
```

Reply:
then you can create driver in this way
`Driver tempDriver = new Driver("Ihor", "11111");`

Reply:
but it's just my recommendation;)

Reply:
you also can create such constructor in Manufacturer)

Reply:
Yes, but the setters do their job well :)